watertap/ui/fsapi.py

```diff
@@ -508,6 +540,21 @@ def export_values(self):
         u = pyo.units
         for key, mo in self.fs_exp.model_objects.items():
             mo.value = pyo.value(u.convert(mo.obj, to_units=mo.ui_units))
+            try:
+                if mo.obj.ub is None:
+                    mo.ub = mo.obj.ub
+                else:
+                    tmp = pyo.Var(initialize=mo.obj.ub, units=u.get_units(mo.obj))
+                    tmp.construct()
+                    mo.ub = pyo.value(u.convert(tmp, to_units=mo.ui_units))
+                if mo.obj.lb is None:
+                    mo.lb = mo.obj.lb
+                else:
+                    tmp = pyo.Var(initialize=mo.obj.lb, units=u.get_units(mo.obj))
+                    tmp.construct()
+                    mo.lb = pyo.value(u.convert(tmp, to_units=mo.ui_units))
+            except Exception as e:
```

Comment:
That `Except` was basically operating as an `if` statement (really the `else` of the `if` statement), as it was getting triggered when the Pyomo object was an expression, because expressions don't have bounds. I just pushed a change to replace that `try/except` with an `if` statement, which checks if the object is an instance of any of the Pyomo Expression classes before trying to set the bounds.